packages/website/src/building-blocs/PageLayout.tsx

```diff
@@ -91,10 +90,12 @@ export const PageLayout: FunctionComponent<PageLayoutProps> = ({
         </div>
     );
 };
-const Content: FunctionComponent<Pick<
-    PageLayoutProps,
-    'code' | 'examples' | 'id' | 'relatedComponents' | 'layout' | 'withPropsTable'
->> = ({code, examples, id, relatedComponents, layout, withPropsTable, children}) => (
+const Content: FunctionComponent<
+    Pick<
+        PageLayoutProps,
+        'code' | 'examples' | 'id' | 'relatedComponents' | 'layout' | 'withPropsTable' | 'propsMetadata'
+    >
+> = ({code, examples, id, relatedComponents, layout, withPropsTable, propsMetadata, children}) => (
```

Comment:
Je prêche pour ma paroisse but this will break soon on react 18 because of the children 😆 
You can wrap the type with PropsWithChildren to help with that 👍 

`const Content: FunctionComponent<PropsWithChildren<
    Pick<
        PageLayoutProps,
        'code' | 'examples' | 'id' | 'relatedComponents' | 'layout' | 'withPropsTable' | 'propsMetadata'
    >
>> `